homeassistant/components/light/zwave.py

```diff
@@ -85,6 +84,10 @@ def __init__(self, values, refresh, delay):
         self._refresh_value = refresh
         self._zw098 = None
 
+        if self._delay:
```

Comment:
I understand you are adding this in order not to make this a breaking change, however the behavior of refresh might still be different than the behaviour of delay so this is potentially a breaking change.

The only advantage is that if we decide to roll this back later - for people who ignored the warning the config will "just work" again.